README.md

```diff
@@ -63,60 +63,77 @@ Optimization simply use Ax.
 - scipy
 - multiple-dispatch
 
+### Prerequisite only for MacOS users:
+Before installing BoTorch, we recommend first manually installing PyTorch, a required dependency of BoTorch. Installing it according to the [PyTorch installation instructions](https://pytorch.org/get-started/locally/) ensures that it is properly linked against MKL, a library that optimizes mathematical computation for Intel processors. This will result in up to an order-of-magnitude speed-up for Bayesian optimization, as at the moment, installing PyTorch from pip does not link against MKL.
 
-##### Installing the latest release
+The PyTorch installation instructions currently recommend:
+1. Install [Anaconda](https://www.anaconda.com/distribution/#download-section). Note that there are different installers for Intel and M1 Macs.
+2. Install PyTorch with `conda install pytorch torchvision -c pytorch`.
```

Comment:
Do we want to add this here? This means that unless we monitor the PyTorch installation instructions this could easily go stale. Ideally we can avoid that by just pointing to the installation instructions. Wdyt?